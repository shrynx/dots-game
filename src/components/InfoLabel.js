// @flow
import styled from 'styled-components';

const InfoLabel = styled.div`
  font-size: 12px;
  text-align: center;
`;

export default InfoLabel;
