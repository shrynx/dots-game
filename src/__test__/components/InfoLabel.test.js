import React from 'react';
import { render } from 'react-testing-library';
import InfoLabel from '../../components/InfoLabel';

it('InfoLabel component should render correctly', () => {
  const { container } = render(<InfoLabel>SPEED</InfoLabel>);
  expect(container).toMatchSnapshot();
});
