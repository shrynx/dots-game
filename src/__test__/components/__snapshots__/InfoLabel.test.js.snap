// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoLabel component should render correctly 1`] = `
<div>
  <div
    class="sc-bdVaJa jwdQHW"
  >
    SPEED
  </div>
</div>
`;
